nimare/resources/references.bib

```diff
@@ -529,3 +529,16 @@ @article{geoffroy2001poisson
   year={2001},
   publisher={Taylor \& Francis}
 }
+
+@article{winkler2016non,
```

Comment:
is this referenced in the permutedOLS class?

Reply:
Good catch! I added that when I defined the `mode` parameter, but later, I removed it in favor of `two_sided` and forgot to remove the citation.